### ISSUE TITLE
Opening different files with same name inside different folders overwrites one of them.

### ISSUE THREAD

**Issue by JonnyCodewalker** [lainsce/quilter#82]
Not sure if the title is accurate, will do more testing later today. 

It seems like when you have multible files with the same name and you switch between them one gets overwritten. 

X/A/file.mb >> test text blablabla
X/B/file.mb >> please don overwrite me

Opening B/file.mb -> closing quilter -> opening A/file.mb closing quilter -> opening B/file.mb and it is overwritten with the content of a (filesize and path shown on the top confirm that).

I suspect that it has to do with the way that it openes the last open file on startup. Seeing the same name it might save it to "update" it, just a guess though.

**Comment by lainsce** [comment]
That's strange, it shouldn't do that. :thinking:
It should see that the paths are different and not update the new one with the contents of the old one.

**Comment by JonnyCodewalker** [comment]
I will see if I can capture it on video or be able to dig more into it to see why it happens.

**Comment by JonnyCodewalker** [comment]
Update: was able to verify the bug occuring in a freshly set up and clean virtual machine (to eliminate possible sideeffects of other programs installed/hardware) after closing and opening the files around 3 times.

**Comment by lainsce** [comment]
Maybe you're closing the file too fast for the system to save it, so I'm going to see if there's a way to make this not occur

**Comment by JonnyCodewalker** [comment]
Well the issue is not that it isn´t saving, but that it overrides :/ will investigate further tomorrow.

**Comment by jayeless** [comment]
I'm experiencing the same issue. I can give two examples of how it's occurred for me:

1. Having never used Quilter before, I decided to open a close-to-empty text file (from Files) to write in it. I decided to open a second file (also from Files) to refer to it while working on this one, but instead of them opening side-by-side as I expected, the file path at the top changes to that of the second file while retaining the content of the first. When I close this file, exiting Quilter, the second file is overwritten with the contents of the first file (as seen through Dropbox's web viewer).
2. About five minutes later, I opened another file (again by clicking on it in Files). According to the file path at the top, it's opened the file I clicked on, but the contents are that of the second file (whose contents I'd restored using Dropbox version history, so I see the contents as they are now, not as they were last time I had Quilter open). When I close it, the contents of this third file have been overwritten by those of the second file.

None of the files had the same name, and the third file was in a separate folder from the first two. (It was in a subfolder of the one the first two are in, though.)

All of this is with Quilter 1.3.6 (the latest version available through AppCenter, if not actually the latest), in elementary OS.
